src/models/vae.jl

```diff
@@ -14,6 +16,16 @@ Safe version of softplus.
 """
 safe_softplus(x::T) where T  = softplus(x) + T(0.000001)
 
+"""
+	init_vamp(pseudoinput_mean,k::Int)
+
+Initializes the VAMP prior from a mean vector and number of components.
+"""
+function init_vamp(pseudoinput_mean, k::Int)
+	T = eltype(pseudoinput_mean)
+	pseudoinputs = T(1) .* randn(T, size(pseudoinput_mean)[1:end-1]..., k) .+ pseudoinput_mean
+	VAMP(pseudoinputs)
```

Comment:
Out of curiosity. Where is VAMP imported from?

Reply:
It's from GenerativeModels.jl